src/statuscake.js

```diff
@@ -2,14 +2,13 @@ const axios = require('axios');
 const debug = require('debug')('statuscake');
 const logError = require('debug')('error');
 const config = require('config');
-const Storage = require('./statuscakedata');
 
 const { username, apikey, baseUrl } = config.statuscake;
 const STATUS_CAKE_BASE_URL = `${baseUrl}?API=${apikey}&Username=${username}`;
 
-const fetchStatuCakeData = () => axios.get(STATUS_CAKE_BASE_URL)
+const fetchStatuCakeData = Storage => axios.get(STATUS_CAKE_BASE_URL)
   .then(({ data: { data: urls } }) => {
-    urls.forEach((item) => {
+    const urlsFormated = urls.map((item) => {
```

Comment:
you can destructure directly inside the map callback 💃 
And return directly the object without assigning to `myNewItem`